lib/code_fund/schemas/click.ex

```diff
@@ -58,13 +60,18 @@ defmodule CodeFund.Schema.Click do
     :property_id,
     :ip,
     :revenue_amount,
-    :distribution_amount
+    :distribution_amount,
+    :redirected_at,
```

Comment:
These actually are not required b/c they are only populated if they actually redirect. (redirected_at and redirected_to_url)